DigitalLearningSolutions.Data.Tests/DataServices/CentresDataServiceTests.cs

```diff
@@ -1,29 +1,30 @@
-﻿namespace DigitalLearningSolutions.Data.Tests.Services

```

Comment:
Could we add the new page to the Automated UI tests (don't mind if it's part of the next ticket as you can probably simulate the whole registration journey to go through all the pages - happy to have a chat about this)?